### PR TITLE
Improve latent diff example

### PR DESCRIPTION
**Pull request by satpalsr**
**Use GPU and improve code**
Original Code:
```
# !pip install diffusers transformers
from diffusers import DiffusionPipeline

model_id = "CompVis/ldm-text2im-large-256"

# load model and scheduler
ldm = DiffusionPipeline.from_pretrained(model_id)

# run pipeline in inference (sample random noise and denoise)
prompt = "A painting of a squirrel eating a burger"
images = ldm([prompt], num_inference_steps=50, eta=0.3, guidance_scale=6).images

# save images
for idx, image in enumerate(images):
    image.save(f"squirrel-{idx}.png")
```

Generates an error (This PR fixes it):
`AttributeError: 'dict' object has no attribute 'images'`

### CODE REVIEW

**Comment by patrickvonplaten** [comment]
Hey @satpalsr, 

The `.images` attribute will be added today! So the code should work then. Hope it's ok to close for now :sweat_smile:

File: README.md [hunk 1]
```diff
@@ -254,20 +254,23 @@ If you want to run the code yourself 💻, you can try out:
...
 
 # load model and scheduler
 ldm = DiffusionPipeline.from_pretrained(model_id)
+ldm = ldm.to(device)
 
 # run pipeline in inference (sample random noise and denoise)
 prompt = "A painting of a squirrel eating a burger"
-images = ldm([prompt], num_inference_steps=50, eta=0.3, guidance_scale=6).images
+with autocast(device):
+    image = ldm([prompt], num_inference_steps=50, eta=0.3, guidance_scale=6)
```
Comment (patrickvonplaten) [inline README.md:270]:
```suggestion
    image = ldm([prompt], num_inference_steps=50, eta=0.3, guidance_scale=6).images[0]
```

File: README.md [hunk 2]
```diff
@@ -254,20 +254,23 @@ If you want to run the code yourself 💻, you can try out:
...
 
-# save images
-for idx, image in enumerate(images):
-    image.save(f"squirrel-{idx}.png")
+# save image
+image['sample'][0].save("squirrel.png")
```
Comment (patrickvonplaten) [inline README.md:273]:
```suggestion
image.save("squirrel.png")
```

**Comment by patrickvonplaten** [comment]
@satpalsr - actually there were a couple of nice improvements. Could change the image API back to before and leave the rest? Happy to merge then :-)

**Comment by satpalsr** [comment]
Thanks for the info. Accepted the suggestions.

**Comment by patrickvonplaten** [comment]
Thanks!

**Pull request merged**